src/lib/components/workflow-actions.svelte

```diff
@@ -206,23 +206,22 @@
             workflowType: workflow.name,
           }),
         )}
+      disabled={workflowCreateDisabled($page)}
       data-testid="start-workflow-button"
     >
       {translate('workflows.start-workflow-like-this-one')}
     </MenuItem>
+    {#if terminateEnabled && next}
+      <MenuDivider />
+      <MenuItem
```

Comment:
If we're no longer showing the tooltip on disabled can we remove `resetDescription`?

Reply:
We still use resetDescription on running workflows for the reset menu item